.eleventy.js

```diff
@@ -160,6 +160,13 @@ module.exports = eleventyConfig => {
 		value1.concat(value2),
 	);
 
+	eleventyConfig.addFilter("lastUpdated", filepath => {
+		const relativePath = path.relative("src/content/blog", filepath);
+		const blogDates = require("./src/_data/blog-dates.json");
+
+		return new Date(blogDates[relativePath]);
```

Comment:
When the entry for a blog post doesn't exist in `blog-dates.json`, this results in: 

![image](https://github.com/user-attachments/assets/47a1d272-ba28-4d76-a71f-aadd04ef8320)

Perhaps we should ensure that in the mentioned case the page shows the publish date here?